tests/api/test_api_simple_flow.py

```diff
@@ -46,7 +46,7 @@ def test_simple_flow(client, document_json_fixture):
 
 def test_add_files_restrictions(client, document_with_file):
     """Test adding file restrictions before dumping object."""
-    res = client.get('https://localhost:5000/documents/10000')
+    res = client.get('https://localhost:5000/documents/10000?resolve=1')
```

Comment:
Can you use `url_for`?

Reply:
Done in all tests! Thanks for the review!